internal/converter/converter.go

```diff
@@ -0,0 +1,57 @@
+// Package converter implements functionality for converting and compressing images.
+package converter
+
+import (
+	"bytes"
+	"fmt"
+	"image"
+	"image/jpeg"
+	"image/png"
+	"io"
+	"mime/multipart"
+)
+
+const (
+	FormatJPEG = "jpeg"
+	FormatJPG  = "jpg"
+	FormatPNG  = "png"
+)
+
+// Converter converts and compresses images.
+type Converter struct{}
+
+// NewConverter creates new converter.
+func NewConverter() *Converter {
+	return &Converter{}
+}
```

Comment:
Do we need a converter type?

Reply:
> Do we need a converter type?

I agree, I went too far with structuring :)